functions.php

```diff
@@ -199,7 +199,7 @@ function gesso_editor_scripts() {
  */
 function gesso_editor_styles() {
 	add_editor_style( 'https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,300;0,400;0,600;0,700;1,300;1,400;1,600;1,700&display=swap' );
-	add_editor_style( get_stylesheet_directory_uri() . '/build/css/editor-styles.css' );
+	add_editor_style( './build/css/editor-styles.css' );
```

Comment:
I'm thinking that it might be best to move to using `get_stylesheet_directory()` . https://developer.wordpress.org/reference/functions/get_stylesheet_directory/

Reply:
@tropicandid since `get_stylesheet_directory()`  returns the full filesystem path we would need to trim off part of it. This would just help to ensure we don't get something weird back without any sort of expected theme path.

Reply:
@tropicandid so actually as I was just thinking about this more the context for this being used is a filesystem call instead of a web path, thus the issue with Basic Auth, so I believe just using the full path is probably perfectly fine and simple enough to use/test?

Reply:
@timnolte So, trying to implement this with get_stylesheet_directory locally proves to not work. I will try to see if i can manipulate how that is returning the server path but so far it is proving to not return what we want locally.

Reply:
@tropicandid OK, this is interesting. After digging into the `add_editor_style()` function [docs](https://developer.wordpress.org/reference/functions/add_editor_style/) it actually does indicate that the path should be relative to the theme root. Also, there is a note in those Docs about the handling of parent & child themes that `add_editor_style()` actually does handle.

> If a child theme is used, both the current child and parent theme directories are tested and both the files with the same relative path are linked with this single call if they are found.

Sorry, I should have checked those docs before sending you down this rabbit hole. Looks like your original fix is the right solution.